src/components/profileConf/profileConfig/index.js

```diff
@@ -0,0 +1,179 @@
+        try {
+          const response = await axios.get(`https://serene-coast-44000.herokuapp.com/users/profile/2`);
+          setUser(response.data);
+        } catch (error) {
+          console.log(error);
+        }
+      });
+  }
+
+  return (
+  <>
+    {chUserName && (
+      <ProfileConf>
+        <motion.div 
+          initial={linkAnimateFrom}
+          animate={linkAnimateTo}
+          transition={{delay: 0.10}}
+          className="configModal">
+          <img
+            className="closeModalButton" 
+            src={closeModalButton}
+            onClick={()=>setChUserName(!chUserName)}
+          ></img>
+          <h2 className="chConf__title">Change User Name</h2>
+          <p className="chConf__Msg">Are you sure you want to change your User Name?</p>
+          <div className="chConf__Options">
+            <button 
+              type="button"
+              className="confirmButton"
+              onClick={()=>{ setChUserName(!chUserName);setChangingUsr(!changingUsr)}}
+            >
+              Yes, Change User Name
+            </button>
+            <button 
+              type="button" 
+              className="cancelButton"
+              onClick={()=>setChUserName(!chUserName)}
+              >
+                Cancel
+            </button>
+          </div>
+        </motion.div>
+      </ProfileConf>)
+    }
+    {chPassword && (
+      <ProfileConf>
+        <motion.div 
+          initial={linkAnimateFrom}
+          animate={linkAnimateTo}
+          transition={{delay: 0.10}}
+          className="configModal">
+          <img
+            className="closeModalButton" 
+            src={closeModalButton}
+            onClick={()=>setChPassword(!chPassword)}
+          ></img>
+          <h2 className="chConf__title">Change Password</h2>
+          <p className="chConf__Msg">Are you sure you want to change your Password?</p>
+          <div className="chConf__Options">
+            <button 
+              type="button"
+              className="confirmButton"
+              onClick={()=> {setChangingPass(!changingPass);setChPassword(!chPassword)}}
+            >
+              Yes, Change password
+            </button>
+            <button 
+              type="button" 
+              className="cancelButton"
+              onClick={()=>setChPassword(!chPassword)}
+            >
+              Cancel
+            </button>
+          </div>
+        </motion.div>
+      </ProfileConf>)
+    }
+    { changingUsr && (
+      <ProfileConf>
+        <motion.div 
+          initial={linkAnimateFrom}
+          animate={linkAnimateTo}
+          transition={{delay: 0.10}}
+          className="changingModal">
+          <img
+            className="closeModalButton" 
+            src={closeModalButton}
+            onClick={()=>setChangingUsr(!changingUsr)}
+          ></img>
+          <h2 className="chConf__title">Change User Name</h2>
+          <form className="changingForm">
```

Comment:
Do you still need to apply the logic in the form, don't you?

Reply:
sip, aun falta la logica, que voy a hacer hoy :v y pues la mando asi ya que de momento no tenemos ni DB xd

Reply:
de hecho el push era mas guiado hacia resolver el filmView con lo del Contex y pues adicionalmente se fueron mis html-css xd por eso lo puse como un "extra" para dejar como doc que donde/cuando se agrega que cosa xd